File: 파이썬/최대공약수와 최소공배수.py
def gcd(a, b):
    return a if b == 0 else gcd(b, a % b)


def solution(n, m):
    a = max(n, m)
    b = min(n, m)
    return [gcd(a, b), n*m // gcd(a, b)]

File: 자바스크립트/최대공약수와 최소공배수.js
function solution(n, m) {
    const a = n > m ? n : m;
    const b = n > m ? m : n;

    const gcd = (a, b) => b ? gcd(b, a % b) : a;

    return [gcd(n, m), n * m / gcd(n, m)];
}